sma/integrationpluginsma.cpp

```diff
@@ -750,3 +750,9 @@ bool IntegrationPluginSma::isModbusValueValid(quint64 value)
     return value != 0xffffffffffffffff;
 }
 
+double IntegrationPluginSma::roundValue(double value, uint digits)
+{
+    uint factor = pow(10, digits);
+    return qRound(value * factor) * 1.0 / digits;
```

Comment:
No, that can't be right. sould rather be

```
uint factor = qPow(10, digits);
return qRound(value * factor) * 1.0 / factor;
```

Reply:
Ups, fixed